MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: PullToRefresh in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about PullToRefresh support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
```

Comment:
Learn here all about the pull to refresh support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/bb1edd32-ab3a-47b9-8975-2f47ac717a32)
Had updated as discussed.